backend/Controllers/MetadataController.cs

```diff
@@ -105,7 +105,13 @@ public async Task<IActionResult> SetExperience([FromBody] SaveExperiencePostReso
 			}
 			if (!ModelState.IsValid)
 				return BadRequest(ModelState.GetErrorMessages());
-			
+
+
+			if (user.UserType < UserType.Municipality || user.MunicipalityName != metadata.MunicipalityName)
```

Comment:
"!=" instead of "<" ?

Reply:
It is < to ensure roles with more privileges than municipality still can add an experience post.